src/modules/admin/Admin.scss

```diff
@@ -0,0 +1,4 @@
+.container {
```

Comment:
We have convention to name first classes in file with component name - much easier after to find this classes via devtools